lib/bolt/inventory/group2.rb

```diff
@@ -41,7 +41,7 @@ def initialize(input, plugins)
 
         @unresolved_targets = {}
         @resolved_targets = {}
-        @targets = Set.new
```

Comment:
This was not used. The commented out unused instance variable is removed in a different PR.